ADRs/Meal inventory updates.md

```diff
@@ -0,0 +1,66 @@
+# Meal Inventory Updates
+
+## Status
+
+_PROPOSED_
+
+## Context and Problem Statement
+
+Farmacy Foods produces meals in one or more central kitchens, delivers those meals to various locations, where they are sold to customers via a smart fridge or point-of-sale system.
+The meals are the central aspect of the business, and there is a big focus on food quality.
+
+### Requirements
+
+* Each meal will be tracked from the point of its creation in the kitchen, throughout delivery, stocking the fridge, and purchase by the customer.
```

Comment:
With Michael's Data Platform the meal is essentially tracked from the order, before it is created.

Reply:
Added